assets/queries/dockerCompose/cpus_not_limited/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "77d3c3ed-9c6f-45e7-96d4-b688dd4842d8",
+    "queryName": "Cpus Not Limited",
+    "severity": "LOW",
+    "category": "Resource Management",
+    "descriptionText": "Check if theres is a cpu priority defined for containers in the docker compose document.",
```

Comment:
```suggestion
    "descriptionText": "CPU limits should be set because if the system has CPU time free, a container is guaranteed to be allocated as much CPU as it requests",
```